examples/06_03_select/src/lib.rs

```diff
@@ -0,0 +1,183 @@
+#![cfg(test)]
+#![feature(async_await)]
+
```

Comment:
Seems a recursion limit error has occurred in this crate. https://travis-ci.com/rust-lang/async-book/builds/122145149#L961